addons/event/report/event_event_templates.xml

```diff
@@ -1,104 +1,241 @@
+                                        <div class="o_event_full_page_ticket_small" t-field="attendee.partner_id"/>
+                                    </div>
                                 </div>
                             </div>
-                            <div class="row">
-                                <div t-field="event.badge_front"></div>
-                            </div>
-                        </div>
-                        <!-- Back -->
-                        <div class="col-6" style="padding-right:7mm; height: 149mm; padding-top: 2mm;">
-                            <div t-field="event.badge_back"/>
                         </div>
-                    </div>
-                    <div class="row">
-                        <!-- Inner left -->
-                        <div class="col-6 text-center" style="padding-right:7mm; border-top:1px dashed black; height: 148mm;">
-                            <div t-field="event.badge_innerleft"/>
-                        </div>
-                        <!-- Inner right -->
-                        <div class="col-6 text-center" style="border-left:1px dashed black; border-top:1px dashed black; height: 148mm; text-center">
-                            <div t-field="event.badge_innerright"/>
+                        <div class="o_event_full_page_ticket_barcode col-2">
+                            <div class="o_event_full_page_ticket_barcode_container">
+                            </div>
                         </div>
                     </div>
-                </t>
+                </div>
+                <div class="page oe_structure"/>
+                <div t-field="event.ticket_instructions" class="o_event_full_page_extra_instructions px-2 pt-3"></div>
             </div>
+        </div>
+    </div>
+</template>
+
+<template id="event_report_full_page_ticket_layout">
+    <!-- Inspired from "external_layout_standard" to get a repeated footer element. -->
+    <div class="article"
+        t-att-data-oe-model="main_object and main_object._name"
+        t-att-data-oe-id="main_object and main_object.id"
+        t-att-data-oe-lang="main_object and main_object.env.context.get('lang')">
+        <main>
+            <t t-out="0"/>
+        </main>
+    </div>
+
+    <div class="row footer o_event_full_page_ticket_footer d-block">
+        <div class="o_event_full_page_ticket_powered_by text-white text-center p-2 w-100">
+            <t t-if="event.organizer_id">
+                <span class="font-weight-bold" t-field="event.organizer_id.name"></span>
+                <span t-if="event.organizer_id.phone" class="pl-3 fa fa-phone"/>
+                <span t-if="event.organizer_id.phone" t-field="event.organizer_id.phone"/>
+                <span t-if="event.organizer_id.email" class="pl-3 fa fa-envelope"/>
+                <span t-if="event.organizer_id.email" t-field="event.organizer_id.email"/>
```

Comment:
email_normalized maybe ? that way we have a standard display containing only email and not `"Raoul Poilvache" <blah@bla.blah>`